app/Plugin/EnvSource/Config/Schema/schema.xml

```diff
@@ -39,6 +39,7 @@
     </field>
     <field name="duplicate_mode" type="C" size="2" />
     <field name="redirect_on_duplicate" type="C" size="1024" />
+    <field name="sp_type" type="C" size="1" />
```

Comment:
For consistency with other enums, this should be size 2.

Reply:
Done